bin/ansible-galaxy

```diff
@@ -513,20 +514,6 @@ def install_role(role_name, role_version, role_filename, options):
         # verify the role's meta file
         meta_file = None
         members = role_tar_file.getmembers()
-        # next find the metadata file
-        for member in members:
-            if "/meta/main.yml" in member.name:
-                meta_file = member
-                break
-        if not meta_file:
-            print "- error: this role does not appear to have a meta/main.yml file."
-            return False
-        else:
-            try:
-                meta_file_data = yaml.safe_load(role_tar_file.extractfile(meta_file))
-            except:
-                print "- error: this role does not appear to have a valid meta/main.yml file."
-                return False
```

Comment:
We already warn for unparseable/unreadable meta/main.yml in `get_role_metadata`.

I only removed the remaining messages following your earlier comments on missing meta/main.yml

> too noisy, no need to warn in this case